assets/javascript/app.js

```diff
@@ -0,0 +1,100 @@
+// * You'll create a trivia form with multiple choice or true/false options (your choice).
+// Done!
+
+// * The player will have a limited amount of time to finish the quiz. 
+// Done!
+
+//   * The game ends when the time runs out. The page will reveal the number of questions that players answer correctly and incorrectly.
+
+// * Don't let the player pick more than one answer per question.
+
+// * Don't forget to include a countdown timer.
+
+
+
+
+var count = 30;
+var interval = setInterval(function(){
+  document.getElementById('timer').innerHTML=count;
+  count--;
+  if (count === 0){
+    clearInterval(interval);
```

Comment:
Within this function I would have a modal pop up when the timer reaches 0 and a button to reset the game.